### ISSUE TITLE
Register Page  UI 

### ISSUE THREAD

**Issue by maverick1910** [PoojaMittal2842/Delhi-Tourism#83]
Create a register page for the Delhi Tourism website. The page should be responsive and the tech stack given for the project should be used. Please send in a picture/video during PR's.

The details should be:
-First Name
-Last Name
-Email
-Phone Number
-Password
-Confirm Password 
-Confirm Password
-Register with Google
-Register with Facebook

Anyone who is willing to work on this, please comment on this issue so I can assign them the task

**Comment by Vedant-Rahane** [comment]
I would like to work on this issue.
@maverick1910 Please assign me this.

**Comment by GulshanKataria** [comment]
@maverick1910 Can I work on the Register page UI? Can you please assign this issue to me as a part of the GSSOC

**Comment by abhishek96k** [comment]
I would like to work on this issue 
as part of the `gssoc`

**Comment by maverick1910** [comment]
Hey @GulshanKataria ,I have assigned the register page to Vedant as he approached me first, if its fine with you I can assign you the login page and some other pages as well , what do you say ??

**Comment by maverick1910** [comment]
> I would like to work on this issue.
> @maverick1910 Please assign me this.

I've assigned you this issue, please follow the basic folder layout which has already been mentioned 

All the best !

**Comment by GulshanKataria** [comment]
@maverick1910 Yes, sir np. Please assign me some other pages

**Comment by maverick1910** [comment]
@GulshanKataria please comment on the recent issue for book pass issue , so that I can assign you that

**Comment by Vedant-Rahane** [comment]
> > I would like to work on this issue.
> > @maverick1910 Please assign me this.
> 
> I've assigned you this issue, please follow the basic folder layout which has already been mentioned
> 
> All the best !

I am working on it.
Thank you !

**Comment by maverick1910** [comment]
> > > I would like to work on this issue.
> > > @maverick1910 Please assign me this.
> > 
> > I've assigned you this issue, please follow the basic folder layout which has already been mentioned
> > 
> > All the best !
> 
> I am working on it.
> Thank you !
> 

Please try completing it by Friday , so that we can move ahead with the other UI iterations 
All the best :)

**Comment by Vedant-Rahane** [comment]
@maverick1910 should we include country field in registration form as tourist can be from outside India.

**Comment by maverick1910** [comment]
> @maverick1910 should we include country field in registration form as tourist can be from outside India.

Sure , add that field as well !

**Comment by Vedant-Rahane** [comment]
@maverick1910 I have created registration page like this please check if there should be any changes.

![registration-page](https://user-images.githubusercontent.com/71424483/111061112-a403c180-84c7-11eb-815d-abcc738aabcb.png)

Also we can modify - 

- Phone no.
 We can have option to select country code, currently we can only add text in it.

 Like this -

![phonecodes](https://user-images.githubusercontent.com/71424483/111061179-0957b280-84c8-11eb-97fb-4c8bd5acdb1f.png)


- Country 
 We can have searchable dropdown list with all country names, currently we can only add text in it.

 Like this -

![country](https://user-images.githubusercontent.com/71424483/111061217-373cf700-84c8-11eb-897c-c494b99a5eda.png)


But for above features we have to use jQuery and other libraries.
What are your thoughts on this?

**Comment by maverick1910** [comment]
This is perfect , you can go ahead and make a PR

**Comment by maverick1910** [comment]
Try to change the buttons at the bottom,it looks like a card which has buttons inside of it, try creating separately

**Comment by maverick1910** [comment]
@Vedant-Rahane also when you make a pr 
Please make it this way

Issue Name-Your Name

**Comment by maverick1910** [comment]
> @maverick1910 I have created registration page like this please check if there should be any changes.
> 
> ![registration-page](https://user-images.githubusercontent.com/71424483/111061112-a403c180-84c7-11eb-815d-abcc738aabcb.png)
> 
> Also we can modify - 
> 
> - Phone no.
>  We can have option to select country code, currently we can only add text in it.
> 
>  Like this -
> 
> ![phonecodes](https://user-images.githubusercontent.com/71424483/111061179-0957b280-84c8-11eb-97fb-4c8bd5acdb1f.png)
> 
> 
> - Country 
>  We can have searchable dropdown list with all country names, currently we can only add text in it.
> 
>  Like this -
> 
> ![country](https://user-images.githubusercontent.com/71424483/111061217-373cf700-84c8-11eb-897c-c494b99a5eda.png)
> 
> 
> But for above features we have to use jQuery and other libraries.
> What are your thoughts on this?
> 
> 

@Vedant-Rahane go ahead with jQuery as well, create a folder for jquery inside assets

**Comment by Vedant-Rahane** [comment]
Ok, I will make PR when registration page is completed.

**Comment by Vedant-Rahane** [comment]
@maverick1910 Check out now -

![resgistration-completed](https://user-images.githubusercontent.com/71424483/111082887-afd79e00-8530-11eb-9064-980ade3a5bee.png)

Should I make PR ?

**Comment by Vedant-Rahane** [comment]
> Should I make PR ?

@maverick1910  Is folder structure correct ?

![folder-check](https://user-images.githubusercontent.com/71424483/111084157-ffb96380-8536-11eb-964a-bc457abdd43a.png)

**Comment by maverick1910** [comment]
>  
> > Should I make PR ?
> 
> @maverick1910  Is folder structure correct ?
> 
> ![folder-check](https://user-images.githubusercontent.com/71424483/111084157-ffb96380-8536-11eb-964a-bc457abdd43a.png)
> 
> 

@Vedant-Rahane yes, you can make the PR now, the directory structure is correct as well

**Comment by Vedant-Rahane** [comment]
@maverick1910 Can you please add level tag ?

**Comment by Vedant-Rahane** [comment]
While adding dark mode my code got comment so the functionality of country name and telephone code is not working also image src is unknown so image is broken .
Should I open new issue and fix this.

![commentregis](https://user-images.githubusercontent.com/71424483/112944663-c1d05800-9150-11eb-96a3-fbd55b48143e.png)
![regiscorrec](https://user-images.githubusercontent.com/71424483/112944682-c98ffc80-9150-11eb-851c-0eeff21d546d.png)
